integrationTest/integration_test.go

```diff
@@ -1016,6 +1030,19 @@ func makeTests() []*TestGroup {
 			tc("Change back to CNAME", cname("foo", "google2.com.")),
 		),
 
+		testgroup("HTTPS",
+			tc("Create a HTTPS record", https("@", 1, "test.com.", "")),
+			tc("Change params of a HTTPS record", https("@", 1, "test.com.", "port=80")),
+			tc("Change target of a HTTPS record", https("@", 1, "www.test.com.", "")),
+			tc("Change priority a HTTPS record", https("@", 10, "test.com.", "")),
```

Comment:
Only change one parameter at a time.

NOTE: I'm not an expert in the HTTPS/SVCB standard. Please verify if these are realistic parameters. I could be way off base!

```suggestion
			tc("Create a HTTPS record", https("@", 1, "test.com.", "port=80")),
			tc("Change HTTPS priority", https("@", 2, "test.com.", "port=80")),
			tc("Change HTTPS target", https("@", 2, ".", "port=80")),
			tc("Change HTTPS params", https("@", 2, ".", "port=99")),
			tc("Change HTTPS params-empty", https("@", 2, ".", "")),
			tc("Change HTTPS all", https("@", 3, "example.com.", "port=100")),
```